.github/workflows/yocto-build-deploy.yml

```diff
@@ -1649,32 +1649,34 @@ jobs:
         run: |
           aws s3 rm "${S3_IMG_URL}"
 
-      - name: Setup AMI test fleet
-        id: ami-test-fleet
+      - name: Setup AMI test Device
+        id: ami-test-device
         env:
           HOSTOS_VERSION: "${{ needs.balena-lib.outputs.os_version }}"
-          AMI_TEST_ORG: testbot
           AMI_TEST_DEV_MODE: true
+          # Pre-made permanent fleet for provisioning AMI's in to for testing
+          AMI_TEST_ORG: ${{ vars.AMI_TEST_ORG }}
+          AMI_TEST_FLEET: ${{ vars.AMI_TEST_FLEET }}
```

Comment:
Will it help to test this PR to codify these vars first, or just set some sane defaults here?